Site/bc_weather_station_dashboard/website/static/JavaScript/precipitation.js

```diff
@@ -0,0 +1,79 @@
+// Function to get current precipitation value
+function getCurrentPrecipitationValue() {
+    // Get precipitation amount
+    var precipitationText = document.getElementById('precipitation').innerText;
+    var match = precipitationText.match(/\d+(\.\d+)?/); 
+    // If number, return number or else return null
+    if (match) {
+        return parseFloat(match[0]);
+    } else {
+        return null;
+    }
+}
+
+// Function to change rain intensity
+function adjustRainIntensity(precipitationValue) {
+    // Calculate intensity
+    var rainAmount;
+    if(precipitationValue === 0) {
+        rainAmount = 50000;
+    } else if(precipitationValue <= 0.5) {
+        rainAmount = 200;
+    } else if(precipitationValue <= 1.0) {
+        rainAmount = 150;
+    } else if(precipitationValue <= 1.5) {
+        rainAmount = 100;
+    } else if(precipitationValue <= 2.0) {
+        rainAmount = 60;
+    } else if(precipitationValue <= 4.0) {
+        rainAmount = 40;
+    } else if(precipitationValue <= 6.0) {
+        rainAmount = 20;
+    } else {
+        rainAmount = 5;
+    }
```

Comment:
Try replacing this big block with this:
`var rainAmount = 100/(precipitationValue+.1);`
It should give you roughly the same values

Reply:
Changed it, good idea